generic3g/ComponentSpecParser/parse_component_spec.F90

```diff
@@ -27,10 +27,34 @@ module function parse_component_spec(hconfig, registry, timeStep, refTime, rc) r
       spec%connections = parse_connections(mapl_cfg, _RC)
       spec%children = parse_children(mapl_cfg, _RC)
 
+      call parse_misc(spec, mapl_cfg, _RC)
+
       call ESMF_HConfigDestroy(mapl_cfg, _RC)
 
       _RETURN(_SUCCESS)
    end function parse_component_spec
 
+   ! TODO - we may want a `misc` section in the mapl section, but
+   ! should wait to see what else goes there.  Or maybe a `test`
+   ! section?
+   
+   subroutine parse_misc(spec, hconfig, rc)
```

Comment:
If we were to keep the `activate` switch localized, this would go in `parse_var_specs`.

Reply:
How is that more localized than in the MAPL section itself?   The distinction seems at best minor, and at worst misleading as it needs to impact variables that are specified with the ACG / source code and not just those in the `states` section.